devito/core/autotuning.py

```diff
@@ -300,7 +300,7 @@ def generate_nthreads(nthreads, args, level):
 
 
 options = {
-    'squeezer': 4,
+    'squeezer': 2,
```

Comment:
WHy?

Reply:
quicker auto-tuning, same result at the end on my machine. Tested on a few examples

Reply:
Is this user modifiable?

Reply:
only manually ATM

Reply:
Then I would keep 4, don't want to hard code too restrictive constants (personal can ignore)